ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -172,6 +186,18 @@ function hasPointError(point: any[]) {
   return point[0] > 180 || point[0] < -180 || point[1] > 90 || point[1] < -90
 }
 
+function getDateErrors(filter: any): Set<string> {
+  const errors = new Set<string>()
```

Comment:
```suggestion
  const errors = new Set<string>()
  const value = deserialize(filter.value)
  if (value !== undefined && relativeTimeError(value)) {
    errors.add('relative time exceeds max value')
  }
  return errors
}
```

This should work too. A bit shorter but also a bit less easy to read, your call.